tests/cli/commands/test_user_command.py

```diff
@@ -177,7 +177,6 @@ def find_by_username(username):
             matches = [u for u in retrieved_users if u['username'] == username]
             if not matches:
                 self.fail("Couldn't find user with username {}".format(username))
-                return None
```

Comment:
I think it's reachable in fact. Simply IntelliJ thinks it's not (because of the "fail" name for some reason). But this is a "normal" method, so it needs # noqa so that IntelliJ's false positive is silenced.

Reply:
Ah no. This one is truly unreachable.